### DISCUSSION TITLE
Separate the template using GitHub Actions and Netlify

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#73] [General]
Hi Mara,

As a newbie, I found quite worried when seeing that there are so many failed build actions, although I deploy via Netlify.

Will the history of actions be cleaner if you separate the template using Netlify, and the template using GitHub Actions? As a current workaround, I [disable the workflow](https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow) in GitHub Actions.

It means that:
In the template using Netlify
    - remove the folder `.github/workflows`
    - add a file `netlify.toml` [^1] [^2]
```yaml
# netlify.toml
[build]
  command = "mkdocs build"
  publish = "site"
```

[^1]: https://docs.netlify.com/configure-builds/file-based-configuration/
[^2]: https://docs.netlify.com/integrations/frameworks/#mkdocs

**Comment by Mara-Li** [comment]
I don't want to create a second repo for the template...
I can add some info about it in the docs.


Maybe I can use two branch? I really dunno how i can avoid this without creating two repo

BTW I choose to quit netlify because they continue to use python 3.8, and a lot of plugin start to stop support 3.8.

**Reply by andrey-jef** [reply]
I get your point.

I gg a bit. And found that Cloudflare Pages still continue to use python 3.7 [^1]. It seems that only GitHub Pages support 3.10 runtime? They are so amazing then.

[^1]: https://developers.cloudflare.com/pages/framework-guides/deploy-an-mkdocs-site/

**Reply by Mara-Li** [reply]
More than this ! Github page support any python version, up to 3.11. Python 3.11 is pretty powerful. Yes, the build time is a little longer than Netlify (~2min than ~1), but I have 3.11 so all plugin can be updated & improved.

**Comment by Mara-Li** [comment]
Hello!
I finally choose to create a separate repo for netlify : https://github.com/ObsidianPublisher/publisher-template-netlify

All contents will be sync between the template, so the update will be applied to the two!
